modules/server/src/main/ts/bedrock/cli/ClOptions.ts

```diff
@@ -43,7 +43,8 @@ export const browser: ClOption = {
     'ie',
     'firefox',
     'firefox-headless',
-    'MicrosoftEdge',
+    'MicrosoftEdge', // ? I doubt this should be here.
```

Comment:
Not only is it necessary, if you say `edge` it aliases back to this
https://github.com/tinymce/bedrock/blob/3912e4e00d04809596a38044947d910dc012787d/modules/server/src/main/ts/bedrock/auto/Driver.ts#L29

It's weird but updating all the myriad places that refer to this as `MicrosoftEdge` is too painful at the moment. So we aliased it to load the edge-chromium driver even though that's not what this normally refers to.

We have two options:
* Leave two edge references in here, which is weird and will probably break if someone tries to use `edge` for local testing
* Add an alias in the remote driver so `MicrosoftEdge` becomes `edge`, which is perhaps even more weird
```suggestion
    'MicrosoftEdge',
```

Reply:
Happy to keep it as s for this release and note it down